transformer_lens/HookedTransformer.py

```diff
@@ -334,6 +334,10 @@ def input_to_embed(
             # keys and queries. See HookedTransformerConfig for details
             residual = embed
             shortformer_pos_embed = None
+        #TODO: alibi embedding doesnt do anything
```

Comment:
Why is this a TODO? Should Alibi do something here?

Reply:
It's no longer needed, deleted.